web/styles/message_row.css

```diff
@@ -938,11 +938,18 @@ of the base style defined for a read-only textarea in dark mode. */
 }
 
 /* Placehold the control buttons, unless there's a failure. */
-.locally-echoed
+.locally-echoed {
     .messagebox:hover
-    .message_controls
-    .message_control_button:not(.failed_message_action) {
-    visibility: hidden;
+        .message_controls
+        .message_control_button:not(.failed_message_action) {
+        visibility: hidden;
+    }
+    /* The actual star icon is not added to the DOM when a
+       message is locally echoed, so we set the equivalent
+       width on its container: 22px at 16px/1em. */
+    .star_container {
+        width: 1.375em;
```

Comment:
Is this width guaranteed to be exactly the same in all cases? I think using a `calc` or CSS variable might be helpful. If it's `1em + 6px`, that doesn't seem like it'd be the same with different font-sizes.

Reply:
Ah, good call. I'd misremembered eliminating the `--message-box-icon-width` variable, so I've placed it here (which does bear out that this width is the same in all cases).